app/src/common/shared/org/mozilla/vrbrowser/browser/engine/EngineProvider.kt

```diff
@@ -25,7 +25,11 @@ object EngineProvider {
 
             builder.crashHandler(CrashReporterService::class.java)
             builder.contentBlocking(ContentBlocking.Settings.Builder()
-                    .antiTracking(ContentBlocking.AntiTracking.AD or ContentBlocking.AntiTracking.SOCIAL or ContentBlocking.AntiTracking.ANALYTIC)
+                    .antiTracking(
+                            ContentBlocking.AntiTracking.AD or
+                            ContentBlocking.AntiTracking.SOCIAL or
+                            ContentBlocking.AntiTracking.ANALYTIC)
```

Comment:
I think we should be using the level from the preferences here. I assume it gets over-written later but we should be starting with the correct anti-tracking values I think.

Reply:
Makes sense, updated